components/iam/pkg/oidc/service.go

```diff
@@ -15,59 +15,56 @@ import (
 
 	"github.com/coreos/go-oidc/v3/oidc"
 	"github.com/gitpod-io/gitpod/common-go/log"
-	"github.com/go-chi/chi/v5"
 	"golang.org/x/oauth2"
 )
 
-type OIDCService struct {
-	Handler chi.Router
-
-	configsById      map[string]*OIDCClientConfig
+type Service struct {
```

Comment:
The struct is used as `oidc.Service`, there's no need to duplicate the OIDC in the name again. This change aligns it with go standard guidance.

Reply:
And similarly for the objects below.